n3fit/src/n3fit/stopping.py

```diff
@@ -384,7 +384,7 @@ def e_best_chi2(self):
     @property
     def stop_epoch(self):
         """Epoch in which the fit is stopped"""
-        return self._history.final_epoch + 1
+        return 0 if self._history.final_epoch is None else self._history.final_epoch + 1
```

Comment:
Why the 0?
Wouldn't  it be more correct the final + 1 regardless?

Reply:
I don't exactly recall the issue I ran into here. But to me it seems 0 allows the user to spot that the final epoch was not even created. Maybe we shouldn't fix this at all, it shouldn't happen

Reply:
Done in 323be1f5702b6533ec249376664aba5243ac3741